QUEST_LV_0100.tsv

```diff
@@ -926,8 +926,8 @@ QUEST_LV_0100_20150317_000921	It's the revelation. {nl} Cherish it and do not sh
 QUEST_LV_0100_20150317_000922	Come on. Go to the Paladin Master. {nl} He will tell you the way you must go.
 QUEST_LV_0100_20150317_000923	$I asked my precious friend Algis to go to Tenet Chapel. {nl} I won't lose to Gesti, but it is important to have some back up plan just in case.
 QUEST_LV_0100_20150317_000924	{memo X}
-QUEST_LV_0100_20150317_000925	In the kingdom, there are several valuable weapons aside from this Divine Sphere. {nl} Those weapons saved many lives on Medis Diena four years ago.
-QUEST_LV_0100_20150317_000926	From now on, I will focus on the Divine Sphere. {nl} Meanwhile, please use your skills so that I won't be disturbed.
+QUEST_LV_0100_20150317_000925	In the kingdom, there are several valuable weapons aside from this Sventas Ragas. {nl} Those weapons saved many lives on Medis Diena four years ago.
```

Comment:
In the kingdom, there are several valuable weapons aside from the Sventas Ragas.